example/src/CoverflowExample.js

```diff
@@ -71,17 +59,17 @@ export default class CoverflowExample extends React.Component<*, State> {
       }
     });
 
-    const translateX = position.interpolate({
+    const translateX = Animated.interpolate(position, {
```

Comment:
and related. 

You may consider using `position.interpolate` with `reanimated` as well

Reply:
Does the method exist? It didn't in the version I'm using at least (expo)